src/bp/ui-studio/src/web/components/Layout/BotUnmountedWarning.tsx

```diff
@@ -0,0 +1,26 @@
+import { Classes, H5, Intent, Position, Toaster } from '@blueprintjs/core'
+import React, { useEffect } from 'react'
+
+export default () => {
+  if (!window.IS_BOT_MOUNTED) {
```

Comment:
Can you invert this condition? The if block is big and adds an unnecessary indentation
```js
 if (window.IS_BOT_MOUNTED) {
  return null
}
```